### PR TITLE
{bio}[foss/2020a] NanopolishComp v0.6.11, nanopolish v0.13.2 w/ Python 3.8.2

### PR DESCRIPTION
**Pull request by deniskristak**
… nanopolish-0.13.2-foss-2020a-Python-3.8.2.eb, Biopython-1.78-foss-2020a-Python-3.8.2.eb

### CODE REVIEW

File: easybuild/easyconfigs/n/NanopolishComp/NanopolishComp-0.6.11-foss-2020a-Python-3.8.2.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    (name, version, {
+        'modulename': 'NanopolishComp',
+        'source_urls': ['https://pypi.python.org/packages/source/n/NanopolishComp'],
+        'checksums': ['753eed9ed93d12b15b693c43382c73379064d86eeef651611fa27abe7f8a44b3'],
+    }),
+]
```
Comment (Micket) [inline easybuild/easyconfigs/n/NanopolishComp/NanopolishComp-0.6.11-foss-2020a-Python-3.8.2.eb:28]: Only one extension, then PythonPackage should be used instead

File: easybuild/easyconfigs/n/NanopolishComp/NanopolishComp-0.6.11-foss-2020a-Python-3.8.2.eb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+sanity_pip_check = True
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
```
Comment (Micket) [inline easybuild/easyconfigs/n/NanopolishComp/NanopolishComp-0.6.11-foss-2020a-Python-3.8.2.eb:35]: For PythonBundle and PythonPackage, this is the default sanity check already, so you can remove it

File: easybuild/easyconfigs/n/NanopolishComp/NanopolishComp-0.6.11-foss-2020a-Python-3.8.2.eb [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+sanity_check_commands = [
+]
```
Comment (Micket) [inline easybuild/easyconfigs/n/NanopolishComp/NanopolishComp-0.6.11-foss-2020a-Python-3.8.2.eb:38]: you can remove this variable

**Review by Micket — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/n/NanopolishComp/NanopolishComp-0.6.11-foss-2020a-Python-3.8.2.eb (hunk 1 above)
Comment (deniskristak) [reply easybuild/easyconfigs/n/NanopolishComp/NanopolishComp-0.6.11-foss-2020a-Python-3.8.2.eb:28]: @Micket pythonpackage will fail with `ERROR: Directory '.' is not installable. Neither 'setup.py' nor 'pyproject.toml' found.`

Comment (deniskristak) [reply easybuild/easyconfigs/n/NanopolishComp/NanopolishComp-0.6.11-foss-2020a-Python-3.8.2.eb:32]: solved, needed to add `sources = [SOURCE_TAR_GZ]`

File: easybuild/easyconfigs/n/NanopolishComp/NanopolishComp-0.6.11-foss-2020a-Python-3.8.2.eb [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+name = 'NanopolishComp'
+version = '0.6.11'
+versionsuffix = "-Python-%(pyver)s"
+
+homepage = 'https://adrienleger.com/NanopolishComp/'
+description = "NanopolishComp is a Python3 package for downstream analyses of Nanopolish output files"
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+sources = [SOURCE_TAR_GZ]
```
Comment (Micket) [inline easybuild/easyconfigs/n/NanopolishComp/NanopolishComp-0.6.11-foss-2020a-Python-3.8.2.eb:12]: please add
```
source_urls = [PYPI_SOURCE]
```

File: easybuild/easyconfigs/n/NanopolishComp/NanopolishComp-0.6.11-foss-2020a-Python-3.8.2.eb [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    ('Python', '3.8.2'),
+    ('tqdm', '4.47.0'),
+    ('SciPy-bundle', '2020.03', versionsuffix),
+]
+
+use_pip = True
+
+options = {'modulename': '%(name)s'}
+
+exts_list = [
```
Comment (Micket) [inline easybuild/easyconfigs/n/NanopolishComp/NanopolishComp-0.6.11-foss-2020a-Python-3.8.2.eb:27]: and then remove the entire `exts_lists` variable

**Review by Micket — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3123.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/2b7755d17e218e3f11979707294a494b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node2656.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/bb93bdcc1ae6ca0ac7a765903b93168b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3501.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/52ad2f73030939ebedb6a1edea3b5316 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @deniskristak!

**Pull request merged**